src/instance.ts

```diff
@@ -211,6 +211,10 @@ function setupBabel(loaderConfig: LoaderConfig): any {
 }
 
 function applyDefaults(configFilePath: string, compilerConfig: TsConfig, loaderConfig: LoaderConfig) {
+    // HACK: adding in a blank initializer to stop loader from crashing, 
+    // see s-panferov/awesome-typescript-loader#190 for more details
+    compilerConfig.raw.compilerOptions.exclude = compilerConfig.raw.compilerOptions.exclude || [];
```

Comment:
I think the crash was in 2.2.4 and not in master so this fix doesn't apply. The line that was crashing isn't even here: compilerConfig.typingOptions.exclude = compilerConfig.typingOptions.exclude || [];